awx/main/models/inventory.py

```diff
@@ -209,6 +209,70 @@ def get_group_children_map(self):
             group_children.add(from_group_id)
         return group_children_map
 
+    def get_script_data(self, data, hostvars=False, show_all=False):
```

Comment:
yeah, so my vote would be to get rid of the `data` param here, and add back in the `host` param, which should have a default of `None`.

Here is an example of the options signature that I would expect this to match:

https://github.com/AlanCoding/Ansible-inventory-file-examples/blob/master/scripts/host_flag.py#L11

There are some other fancy things like --tree that `ansible-inventory` provide, and it would be the ideal model for the kwargs structure, although, of course, no need to add anything we're not using.